docs/source/conf.py

```diff
@@ -33,6 +33,7 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.viewcode',
+    'sphinx.ext.napoleon',
```

Comment:
@lsetiawan I forgot that `napoleon`, the extension for the numpy/scipy and google style docstrings, is builtin and we still need to enable it.

Reply:
Thanks for catching that! :)